src/idiomatic/leveraging-the-type-system/extension-traits/extending-other-traits.md

```diff
@@ -0,0 +1,86 @@
+---
+minutes: 15
+---
+
+# Extending Other Traits
+
+Extension traits can attach new methods to _all_ implementors of a given trait:
```

Comment:
Like on the other slide, I'd suggest to start by explaining the desired result first.

Reply:
Done in 771a37a8